libs/site-base/internal/webpack/webpack.dev.babel.js

```diff
@@ -9,97 +9,114 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
-    // Costs too much time when debugging
-    tsCompilerOptions: {
-      noUnusedLocals: false,
-      noUnusedParameters: false,
-    },
-
-    // Add development plugins
-    plugins: [
-      new ErrorOverlayPlugin(),
-      new webpack.ProgressPlugin(),
-      new webpack.HotModuleReplacementPlugin(), // Tell webpack we want hot reloading
-      new HtmlWebpackPlugin({
-        inject: true, // Inject all files that are generated by webpack, e.g. bundle.js
-        template: 'src/index.html',
-        favicon: 'src/images/favicon.png',
-      }),
-      new CircularDependencyPlugin({
-        exclude: /a\.js|node_modules/, // exclude node_modules
-        failOnError: false, // show a warning when there is a circular dependency
-      }),
-    ],
+  // Add development plugins
+  plugins: [
+    new ErrorOverlayPlugin(),
+    new webpack.ProgressPlugin(),
+    new webpack.HotModuleReplacementPlugin(), // Tell webpack we want hot reloading
+    new HtmlWebpackPlugin({
+      inject: true, // Inject all files that are generated by webpack, e.g. bundle.js
+      template: 'src/index.html',
+      favicon: 'src/images/favicon.png',
+    }),
+    new CircularDependencyPlugin({
+      exclude: /a\.js|node_modules/, // exclude node_modules
+      failOnError: false, // show a warning when there is a circular dependency
+    }),
+  ],
 
-    // Emit a source map for easier debugging
-    // See https://webpack.js.org/configuration/devtool/#devtool
-    devtool: 'cheap-module-source-map',
+  // Emit a source map for easier debugging
+  // See https://webpack.js.org/configuration/devtool/#devtool
+  devtool: 'cheap-module-source-map',
 
-    performance: {
-      hints: false,
-    },
+  performance: {
+    hints: false,
   },
-    mockOptions
-))
+};
 
 
 function getMockOptions() {
-  const mocksFolder = path.join(__dirname, '..', '..', '..', '..', '__mocks__');
   return {
-    rules: [
-      // Allow ts-loader to parse mocks
-      {
-        test: /\.ts(x?)$/,
-        include: mocksFolder,
-        use: {
-          loader: 'ts-loader',
-          options: {
-            configFile: path.join(mocksFolder, '..', 'tsconfig.base.json'),
-            transpileOnly: true,
-            experimentalWatchApi: true,
+    module: {
+      rules: [
+        // Allow ts-loader to parse mocks
+        {
+          test: /\.ts(x?)$/,
+          include: mocksFolder,
+          use: {
+            loader: 'ts-loader',
+            options: {
+              configFile: path.join(mocksFolder, '..', 'tsconfig.base.json'),
+              transpileOnly: true,
+              experimentalWatchApi: true,
+            },
           },
-        },
-      }
-    ],
+        }
+      ],
+    },
     // Re-use our jest mocks inside our website (neat, huh?)
     resolve: {
       modules: [mocksFolder]
     }
   }
 }
+
+function getDevLiveOptions() {
+  return {
+    experiments: {
+      topLevelAwait: true,
+    },
+    // Re-direct store to js file
```

Comment:
we don't actually mock devlive, we just redirect queries to account/store to our devlive-specific version